Cargo.toml

```
@@ -1,26 +1,31 @@
 [package]
 name = "ui_test"
-version = "0.2.0"
+version = "0.3.0"
 edition = "2021"
 license = "MIT OR Apache-2.0"
 description = "A test framework for testing rustc diagnostics output"
 rust-version = "1.63"
 
 [lib]
-test = true # we have unit tests
+test = true     # we have unit tests
 doctest = false # but no doc tests
 
 [dependencies]
 rustc_version = "0.4"
 colored = "2"
 # Features chosen to match those required by env_logger, to avoid rebuilds
-regex = { version = "1.5.5", default-features = false, features = ["perf", "std"] }
+regex = { version = "1.5.5", default-features = false, features = [
+    "perf",
+    "std",
+] }
```

```suggestion
regex = {
    version = "1.5.5",
    default-features = false,
    features = ["perf", "std"],
}
```
Seems more readable to me? Same for `color-eyre` below.

---

that isn't valid unfortunately. I used `[dependencies.regex]` instead